src/java.desktop/share/classes/javax/swing/plaf/synth/SynthSplitPaneUI.java

```diff
@@ -265,6 +269,16 @@ protected Component createDefaultNonContinuousLayoutDivider() {
         return new Canvas() {
             public void paint(Graphics g) {
                 paintDragDivider(g, 0, 0, getWidth(), getHeight());
+                if(!isContinuousLayout() && getLastDragLocation() != -1) {
+                    Dimension      size = splitPane.getSize();
```

Comment:
We can eliminate creation on new variable and use splitPane.getWidth/Height() as it is done at many places in this file.

Reply:
ok..removed..